core/src/util/usFrameworkFactory.cpp

```diff
@@ -0,0 +1,40 @@
+/*=============================================================================
+
+  Library: CppMicroServices
+
+  Copyright (c) The CppMicroServices developers. See the COPYRIGHT
+  file at the top-level directory of this distribution and at
+  https://github.com/saschazelzer/CppMicroServices/COPYRIGHT .
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+=============================================================================*/
+
+#include "usFrameworkFactory.h"
+
+US_BEGIN_NAMESPACE
+
+FrameworkFactory::FrameworkFactory(void) 
+{
+}
+
+FrameworkFactory::~FrameworkFactory(void) 
+{
+}
+
+Framework* FrameworkFactory::newFramework(std::map<std::string, std::string> configuration) 
```

Comment:
I agree, we should add `static const std::string` members to the `Framework `class defining the config options known to the framework.